poc/src/main/scala/poc/frameworks/controller/TestRestEndpoint.scala

```diff
@@ -0,0 +1,44 @@
+package poc.frameworks.controller
+
+import net.liftweb.json.{Extraction, JsonParser, compactRender}
+import org.http4s._
+import org.http4s.dsl._
+import poc.frameworks.dao.{MockUserDao, UserDao}
+import poc.frameworks.model.external.{LoginRequest, ServiceResponse, User}
+import poc.frameworks.service.UserService
+import poc.frameworks.util.JsonFormat
+import poc.frameworks.validation.userValidation
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scalaz.{Kleisli, Reader}
+
+/**
+  * Created by arunavas on 28/9/17.
+  */
```

Comment:
Added this test controller just for poc purpose that multiple controller can be added as service. Later, we'd have entity based controllers like UserController, GroupController etc.